source/administration/identity-access-management/oidc-access-management.rst

```diff
@@ -59,7 +59,11 @@ MinIO provides an example Go application
 :minio-git:`web-identity.go <minio/blob/master/docs/sts/web-identity.go>` that
 handles the full login flow.
 
-OIDC users can alternatively create :ref:`access keys <minio-idp-service-account>` associated to their AD/LDAP user. Service accounts are long-lived credentials which inherit their privileges from the parent user. The parent user can further restrict those privileges while creating the access keys. To create a new access keys, log into the :ref:`MinIO Console <minio-console>` using the OIDC-managed user credentials. From the :guilabel:`Identity` section of the left navigation, select :guilabel:`Service Accounts` followed by the :guilabel:`Create access keys +` button.
+OIDC users can alternatively create :ref:`access keys <minio-idp-service-account>`. 
+Access Keys are long-lived credentials which inherit their privileges from the parent user. 
+The parent user can further restrict those privileges while creating the access keys. 
+To create a new access keys, log into the :ref:`MinIO Console <minio-console>` using the OIDC-managed user credentials. 
```

Comment:
`To create a new access key` ... can't be plural.
Or, `To create new access keys`